### ISSUE TITLE
Fix for incomplete buttpress/buttrelease declaration.

### ISSUE THREAD

**Issue by LordOfMice** [davidebreso/ctmouse#4]
Fix from **BolenB**.

buttpress    BUTTLASTSTATE <>
buttrelease    BUTTLASTSTATE <>
->
buttpress    BUTTLASTSTATE <>,<>,<>
buttrelease    BUTTLASTSTATE <>,<>,<> 

and file with fix attached:
[ctmouse_fix.zip](https://github.com/user-attachments/files/18121895/ctmouse_fix.zip)

**Comment by ecm-pushbx** [comment]
Do you intend to create three instances of the BUTTLASTSTATE structure per line? That's what happens when I assemble a small test case with MASM v5.10 (gratis binary from the 2024 April release of MS-DOS v4). To create a single instance with all default values one empty pair of angle brackets suffices.

Test case:

```
        FOO segment

BUTTLASTSTATE struc
counter dw ?
lastrow dw ?
lastcol dw ?
BUTTLASTSTATE ends

        db 0
buttpress1      BUTTLASTSTATE <>
        db 1
buttpress2      BUTTLASTSTATE <>, <>, <>
        db 2

        FOO ends

end
```

**Comment by davidebreso** [comment]
Looking at the code, it seems the code expects three instances of the BUTTLASTSTATE structure. `buttpress` and `buttrelease` are arrays where index 0 = left, 1 = right and 2 = middle mouse button. 
This is also a better fix to issue #1

**Comment by davidebreso** [comment]
Closed by commit [daa6070](https://github.com/davidebreso/ctmouse/commit/daa6070be9b14b8bd36027d3c9dcafe8ce3019ca)
